specification/sql/resource-manager/readme.md

```diff
@@ -1174,7 +1174,7 @@ input-file:
 - ./Microsoft.Sql/preview/2020-11-01-preview/ElasticPoolOperations.json
 - ./Microsoft.Sql/preview/2020-11-01-preview/EncryptionProtectors.json
 - ./Microsoft.Sql/preview/2020-11-01-preview/FirewallRules.json
-- ./Microsoft.Sql/preview/2020-11-01-preview/InstancePools.json
+- ./Microsoft.Sql/preview/2023-05-01-preview/InstancePools.json
```

Comment:
please remove this line and put 2023-05-01-preview in the bottom of V5 section (by alphabetical order)